blueprints/component/template.hbs

```diff
@@ -0,0 +1 @@
+\{{yield}}
```

Comment:
could we have `component-test.ts` file created inside same folder? (component test references could be found in /tests/`